src/wallet/wallet.h

```diff
@@ -1041,6 +1043,8 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
     void MarkDirty();
     bool UpdateNullifierNoteMap();
     void UpdateNullifierNoteMapWithTx(const CWalletTx& wtx);
+    void UpdateSaplingNullifierNoteMapWithTx(CWalletTx& tx);
+    void UpdateSaplingNullifierNoteMapForBlock(const CBlock* pblock);
```

Comment:
Does it make sense to have these be private methods?

Reply:
Inherited from Sprout so not changing for now, unless there is a need to?